schunk_lwa4d/launch/robot.launch

```diff
@@ -14,13 +14,11 @@
   <node ns="arm" name="driver" pkg="canopen_motor_node" type="canopen_motor_node" output="screen" clear_params="true" launch-prefix="">
      <rosparam command="load" file="$(find schunk_lwa4d)/config/can0.yaml" />
      <rosparam command="load" file="$(find schunk_lwa4d)/config/arm_driver.yaml" />
-     <!--remap from="joint_states" to="/joint_states"/-->
   </node>
 
   <!-- controllers -->
   <rosparam ns="arm" command="load" file="$(find schunk_lwa4d)/config/arm_controller.yaml" />
-  <rosparam ns="arm" command="load" file="$(find schunk_lwa4d)/config/joint_limits.yaml"/>
-  <rosparam ns="arm" command="load" file="$(find schunk_lwa4d)/config/arm_cartesian_controller.yaml"/>
```

Comment:
is it right? why did you commit the changes on arm_cartesian_controller.yaml and joint_limits.yaml if you are not loading the files?  :confused: